python/dgl/utils/filter.py

```diff
@@ -0,0 +1,69 @@
+
+"""Utilities for finding overlap or missing items in arrays."""
+
+from .._ffi.function import _init_api
+from .. import backend as F
+
+
+class Filter(object):
+    """Class used to either find either find the subset of ids that are in this
```

Comment:
"either find either find".  Also we usually capitalize "ids" as "IDs".

Reply:
Fixed in 
fe0bc66.